### DISCUSSION TITLE
Menyimpan data train dan data validation pada image classification

### DISCUSSION THREAD

**Discussion by vidric** [teamalgoritma/community#633] [Q&A] [answered]
Bagaimana cara menyimpan data train dan validation hasil generate yang dihasilkan fungsi image_data_generator() dan flow_images_from_directory() ? saya coba ke RDS, hasilnya tidak bisa dibaca

**Comment by finesaaa** [comment, marked as answer]
Dear @vidric , untuk melakukan hal itu memang diperlukan cara khusus yaitu dapat merujuk referensi berikut https://stackoverflow.com/questions/68505932/how-to-save-augmented-images-using-imagedatagenerator-and-flow-from-directory-in

Khususnya dapat diperhatikan pada bagian berikut ini
```
gen.flow_from_dataframe( group,  x_col='filepaths', y_col=None, target_size=(128,128), class_mode=None,
                                        batch_size=1, shuffle=False, save_to_dir=target_dir, save_prefix='aug-',
                                        save_format='jpg')
```
